tslint.json

```diff
@@ -62,7 +62,7 @@
 			"variable-declaration": "onespace"
 		} ],
 		"use-strict": false,
-		"variable-name": [ true, "check-format", "allow-leading-underscore", "ban-keywords" ],
+		"variable-name": [ true, "check-format", "allow-leading-underscore", "ban-keywords", "allow-pascal-case" ],
```

Comment:
Funny that they chose to make this something to explicitly allow, and that they chose pascal rather than camel. :)